tex/alg-geom/sheaves.tex

```diff
@@ -37,8 +37,24 @@ \section{Pre-sheaves}
 Recall that $\OO_V$ took \emph{open sets of $V$} to \emph{rings},
 with the interpretation that $\OO_V(U)$ was a ``ring of functions''.
 
+Recall from \Cref{sec:sheaf_regular_functions} that $\OO_V$, as a set, consist
```

Comment:
try to give some motivation on why we care about rational functions on Zariski open set (because they're *naturally* defined on Zariski open sets!)

Not sure how clearer it could be this way, unfortunately I don't have a convenient person who have not studied algebraic geometry around to test.